src/icons/icons.stories.html

```diff
@@ -3,18 +3,18 @@ <h2>Icons</h2>
   <section>
     <div>
       <div>omrs-icon-visibility</div>
-      <svg role="img">
+      <svg class="omrs-icon" role="img">
```

Comment:
you now have to put a css class on the outer svg. I looked at options for not requiring that, but this is the way to make the sizing be correct.

Reply:
Works for now.